mw-webapp/src/component/tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,41 @@
+.wrapper {
+  position: relative;
+}
+
+.target {
+  cursor: pointer;
+
+  &:hover + .tooltip {
+    display: block;
+  }
+}
+
+.tooltip {
+  display: none;
+  position: absolute;
+  overflow: hidden;
+}
+
+.top {
+  transform: translateY(-102%) translateX(-50%);
```

Comment:
now your surrogate "margin" will be 2%  (translate), it depends on content and will be look not consistent, better to use absolute values in case like this, agreed?

Reply:
I agree, correct